### PR TITLE
Add comment about no short-circuit evaluation in if statements.

### PR DESCRIPTION
**Pull request by baldersheim**
@bratseth @geirst @havardpe @kkraune PR

### CODE REVIEW

File: en/reference/ranking-expressions.html [hunk 1]
```diff
@@ -157,8 +157,22 @@ <h2 id="the-if-function">The if function</h2>
...
+<em>expensive_expr</em> will always be executed even if <em>cheap_expr</em> is below 1.
+</p>
+<p>
+The above should instead be written as
+<pre>
+if (cheap_expr >= 1, if (expensive_expr > 1, even_more_expensive_expr, -1.0), -1.0)
+</pre>. Order the nested ifs with the cheapest one as the outer one with increasing cost as you nest down.
+If the expressions are already computed it does not matter, but explicit nesting is safer.
+</p>
+Note that there is <em>no short-circuit</em> evaluation. Given <pre>if (cheap_expr >= 1 && expensive_expr > 1, even_more_expensive_expr, -1.0)</pre>
```
Comment (bratseth) [inline en/reference/ranking-expressions.html:175]:
```suggestion
```

**Review by bratseth — CHANGES_REQUESTED** [review]
👍 just remove the duplicate line

**Review by geirst — APPROVED** [review]
👍

**Comment by jobergum** [comment]
Thanks for adding this to documentation!

**Comment by baldersheim** [comment]
I will hold merge until we conclude on adding short-circuit evaluation.

File: en/reference/ranking-expressions.html [hunk 2]
```diff
@@ -157,8 +157,21 @@ <h2 id="the-if-function">The if function</h2>
     <tr><td>~=</td> <td>Approximately equal</td></tr>
     <tr><td>&gt;=</td> <td>Greater than or equal</td></tr>
     <tr><td>&gt;</td> <td>Greater than</td></tr>
+    <tr><td>&&</td> <td>Logical AND - No short-circuit evaluation.</td></tr>
```
Comment (kkraune) [inline en/reference/ranking-expressions.html:160]:
```suggestion
    <tr><td>&amp;&amp;</td> <td>Logical AND - No short-circuit evaluation.</td></tr>
```

**Review by kkraune — APPROVED** [review]

**Comment by bratseth** [comment]
Implementet instead, closing.

**Pull request closed without merging**